setup.py

```diff
@@ -21,8 +21,12 @@
 from version import version
 from pkg_resources import parse_version, require, DistributionNotFound
 
+
 def check_provided(distribution, min_version, max_version=None, optional=False):
     min_version = parse_version(min_version)
+    if isinstance(min_version, tuple):
+        raise RuntimeError("Newer version of setuptools required. Update by running "
```

Comment:
I think the version that added PEP-440 is 8.0 http://setuptools.readthedocs.io/en/latest/history.html#id223. Could you 1) verify that 8.0 behaves as desired, 2) verify that a version before 8.0 does not behave as desired, i.e. returns a tuple and 3) mention 8.0 as the minimally required version in this error message?